bower.json

```diff
@@ -31,7 +31,7 @@
     "angular-local-storage": "~0.2.1",
     "datalib": "^1.1.12",
     "vega-lite-ui": "^0.7.9",
-    "vega-lite": "^0.7.1"
+    "vega-lite": "^0.7.2"
```

Comment:
Don't bother about changing this every single time. As long as we use the latest version, we should be fine.